pkg/disk/disk.go

```diff
@@ -43,8 +43,8 @@ func SameDisk(di1, di2 Info) bool {
 		return false
 	}
 
-	// returns true only if Used, Free and number of free
-	// inodes are same, then its the same disk.
-	return di1.Used == di2.Used && di1.Free == di2.Free &&
-		di1.Ffree == di2.Ffree
+	// returns true only if Used, Free are same, then its the same disk.
+	// we are deliberately not using free inodes as that is unreliable
+	// due the fact that Ffree can vary even for temporary files
+	return di1.Used == di2.Used && di1.Free == di2.Free
```

Comment:
Looks likely to give both false positives (idle disks) and negatives (samples taken while disk was in use). 

Is this the best we can do?

Reply:
Yes this is the best we can, we have to cater for containerized environments as well. We had multiple variations before but none worked for containerized environments.

We can't add more stuff as any more stuff added doesn't work in container environments.